feat(arithmetic): Add docs for the new equations feature (#3865)
getsentry/sentry-docs: src/docs/product/discover-queries/query-builder.mdx

### Cell Filters

Each cell in the table will have an ellipsis appear on hover. This will open a context menu with additional filtering capabilities depending on the value type. For example, you can selectively target a transaction to the search conditions bar by clicking on "Add to filter".

![Menu showing additional options to add to filter or exclude from filter](./discover-table-cell.png)

### Adding Equations

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> You can also add an equation to your query using the query's table columns as equation variables. The result of the equation is displayed in the results table. Learn more in [Adding Query Equations](query-equations/).

Reviewer: I'm good with simplifying this a bit, but suggest the following:

```suggestion
You can also add equations to your query using the table columns as equation variables, and the results will be displayed in the query results table. Learn more in [Adding Query Equations](query-equations/).
```
